quandl/model/model_base.py

```diff
@@ -1,21 +1,26 @@
 class ModelBase(object):
-    def __init__(self, raw_data, **options):
-        self.raw_data = raw_data
+    def __init__(self, code, raw_data=None, **options):
+        self.code = code
+        self._raw_data = raw_data
```

Comment:
The reason I renamed it to `_raw_data` is cause they should really be calling `__get_raw_data__()` method to get the raw data. Even in this case it's not a great idea to call the directly however if we want to expose it that would be the one to rename.